whirlpool/beer_fridge.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+import aiohttp
+
+from .appliance import Appliance
+
+LOGGER = logging.getLogger(__name__)
+
+SETTING_TEMP = "Refrigerator_OpSetTempPreset"
+SETTING_DISPLAY_LOCK = "Sys_OpSetControlLock"
+SETTING_TURBO_MODE = "Sys_OpSetMaxCool"
+
+TEMP_MAP = {
+    -4: 12,
+    -2: 11,
+    0: 10,
+    3: 9,
+    5: 8,
+}
+
+
+class BeerFridge(Appliance):
+    def __init__(self, backend_selector, auth, said, session: aiohttp.ClientSession):
+        Appliance.__init__(self, backend_selector, auth, said, session)
+
+    def get_current_temp(self, real_temp: bool = None):
+        if real_temp:
+            reversed_temp_map = {v: k for k, v in TEMP_MAP.items()}
+            return str(reversed_temp_map[int(self.get_attribute(SETTING_TEMP))])
+        return int(self.get_attribute(SETTING_TEMP))
+
+    async def set_temp(self, temp: int):
+        if 8 <= temp <= 12:
+            await self.send_attributes({SETTING_TEMP: str(temp)})
+
+    async def set_especific_temp(self, temp: int):
+        allowed_temps = [-4, -2, 0, 3, 5]
+        if temp not in allowed_temps:
+            raise ValueError(
+                f"Invalid temperature: {temp}. Allowed values are {allowed_temps}."
+            )
+        await self.send_attributes({SETTING_TEMP: str(TEMP_MAP[temp])})
+
```

Comment:
```suggestion
```
Since we have `set_temp` already, can we remove this one since it duplicates behavior?